crates/biome_fs/src/fs/os.rs

```diff
@@ -244,9 +244,21 @@ fn handle_dir_entry<'scope>(
             Some(
                 "package.json"
                     | "package-lock.json"
+                    | "package-lock.yaml"
+                    | ".npmrc"
+                    | "npm-shrinkwrap.json"
+                    | "yarn.lock"
+                    | "pnpm-lock.yaml"
+                    | "pnpm-workspace.yaml"
+                    | "composer.json"
```

Comment:
Out of curiosity, which does create this file?

Reply:
`composer.json` maybe created on php project.

Reply:
- `yarn.lock` is for `yarn`
- `pnpm-lock.yaml` and `pnpm-workspace.yaml` for `pnpm`
- `composer.json` is for PHP

Does that answer your question?

Reply:
`composer.json` is the equivalent of `packages.json` in the PHP world. It is managed by [Composer](https://getcomposer.org)